airflow-core/src/airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -744,7 +744,6 @@ def post_clear_task_instances(
     common_params = {
         "dry_run": True,
         "task_ids": task_ids,
-        "dag_bag": dag_bag,
```

Comment:
Was this not used?

Reply:
It was only used in `include_dependent_dags=True`, and when a task is depended by another dag using `ExternalTaskMarker`. The dag bag is used to get the dependent dags. Since ti now only reads from db, we don’t (shouldn’t) use a dag bag here.